src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -87,6 +91,7 @@ function MoneyRequestView({betas, report, parentReport, policyCategories, should
 
     const isSettled = ReportUtils.isSettled(moneyRequestReport.reportID);
     const canEdit = ReportUtils.canEditMoneyRequest(parentReportAction);
+    const shouldShowBillable = Permissions.canUseTags(betas) && !lodashGet(policy, 'disabledFields.defaultBillable', true);
```

Comment:
This condition should be:
```suggestion
    const shouldShowBillable = isPolicyExpenseChat && Permissions.canUseTags(betas) && (transactionBillable || !lodashGet(policy, 'disabledFields.defaultBillable', true));
```

From the doc:
> If the billable flag is turned off entirely (on a policy that previously had it enabled), we’ll: Hide the `Billable` flag on any requests (both unpaid or paid), where the flag was not toggled on. 

So we need to show the row if it was toggled on for an expense prior to the field being disabled at the policy level.

You'll also need to move this down a few lines to take advantage of `isPolicyExpenseChat`